_data/banking.yml

```
@@ -773,9 +773,9 @@ websites:
   - name: Openbank
     url: https://www.openbank.es
     img: openbank.png
-    twitter: openbank_es
-    facebook: openbank.es
-    lang: es
+    tfa:
+      - sms
+    doc: https://www.openbank.es/en/security
```

I assume you specifically refer to the PSD2 page.
```suggestion
    doc: https://www.openbank.es/en/security#seguridad-3-tab-4
```

---

Although this url displayed when hovering the "PSD2" tab, it is actually not an anchor. Using this address on its own will not switch to "PSD2". (Tested with Opera, Chrome, Firefox)